index.md

```diff
@@ -8,19 +8,23 @@ Contribute on github: <{{ site.github.repository_url }}>
 
 <div class="bigtable">
 <table>
-  {% for row in site.data.clients %}
-    {% if forloop.first %}
+  <thead>
     <tr>
-      {% for pair in row %}
-        <th>{{ pair[0] }}</th>
+      {% for feature in site.data.spec %}
+        <th>{{ feature[1] }}</th>
       {% endfor %}
     </tr>
-    {% endif %}
-
-    {% tablerow pair in row %}
-      {{ pair[1] }}
-    {% endtablerow %}
-  {% endfor %}
+  </thead>
+  <tbody>
+    {% for client_hash in site.data.clients %}
+      {% assign client = client_hash[1] %}
+        <tr>
+          {% for feature in client %}
+            <td>{{ feature[1] }}</td>
+          {% endfor %}
+        </tr>
+    {% endfor %}
```

Comment:
(my fault) this is ignoring the keys now and assuming all yml files have the same structure/order.

Need to find a way to do something like this instead.

```
    {% for client_hash in site.data.clients %}
      {% assign client = client_hash[1][site.data.spec] %}
      <tr>
        {% for spec_feature in site.data.spec %}
          {% assign client_feature = client[spec_feature[0]][1] %}
          <td>{{ client_feature }}</td>
        {% endfor %}
      </tr>
    {% endfor %}
```

Reply:
Reverted this. I should not try to be so clever.